scripts/west_commands/boards.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import collections
+import os
+import re
+import shutil
+import subprocess
+import sys
+import textwrap
+
+from west import log
+from west import cmake
+from west.configuration import config
+from west.commands import WestCommand
+from cmake import run_cmake
+
+BOARDS_DESCRIPTION = '''\
+Display information about boards
+'''
+
+class Boards(WestCommand):
+
+    def __init__(self):
+        super(Boards, self).__init__(
```

Comment:
My fault as I didn't know this myself until I had used the Python 2 way of doing things in other west commands, but the Python 3 way to write this is just `super().__init__(...)`. Mind changing it here so we don't add more of the old style to the code base?

Reply:
I wondered several times about this but incorrectly assumed that you knew something I didn't :)